lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -216,6 +226,10 @@ def cvv_result(response)
         response['source'] && response['source']['cvv_check'] ? CVVResult.new(response['source']['cvv_check']) : nil
       end
 
+      def use_network_token(response)
+        response['source'] && response['source']['payment_account_reference'].present?
```

Comment:
`payment_account_reference` is a response field unique to a network token source payment method.